fix android view [tablet not included]

ItzKazuki opened the pull request:
check dulu @0rcaaa

ItzKazuki committed:
fix android view [tablet not included]

ItzKazuki commented:
Belom kelar, card babi ounya margin/padding right anjg

Commit:
Merge branch 'android-view' of https://github.com/ItzKazuki/School-web into android-view

0rcaaa commented:
card ganti kek kemaren aj nu, trus jadi karosel di semua media, biar pas upload  gk kebawah

ItzKazuki commented:
![Uploading image.png…]()
hmmmm

ItzKazuki committed:
format beberapa kode
agar mudah di baca, menghilangkan scrollbar, fix achievement id, dan news ny belum selesai ;)

Commit:
remake news section

ItzKazuki committed:
fix conflic with update by 0rcaaa

ItzKazuki commented:
![image](https://github.com/ItzKazuki/School-web/assets/79141593/3abccb9c-c2a0-4615-a773-b2489d09e51c)
loh ini kemana

ItzKazuki commented:
oh beda icon, bntr

ItzKazuki committed:
fix icon

ItzKazuki committed:
fix dot terlau keatas brohh

ItzKazuki commented:
no update ?????

0rcaaa commented:
Nah man, i got migraine already. Tomorrow for sure

Commit:
fix news section (desktop)

Commit:
Merge branch 'android-view' of https://github.com/ItzKazuki/School-web into android-view

Commit:
fix news section (desktop)

0rcaaa commented:
cek nu

0rcaaa committed:
Merge branch 'main' into android-view

ItzKazuki commented:
![image](https://github.com/ItzKazuki/School-web/assets/79141593/f93ff84d-0480-4a92-9fae-e5a66b3fa314)
hmmmmmm

ItzKazuki committed:
fix android view bug
